CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## UNRELEASED
+
+### Added
+
+- Source `regactions.registry` [#115]
+- Field with path `tech_data.engine.standarts.emission.euro` [#115]
+- Field with path `additional_info.vehicle.segment.euro.code` [#115]
+- Field with path `additional_info.vehicle.segment.euro.description` [#115]
+- Field with path `registration_actions.items[].actuality.date` [#115]
+
```

Comment:
Add mentions that some fields also fillable by new source, e.g. 
```
- Field with path `tech_data.engine.standarts.emission.euro` also fillable by `regactions.registry` source [#115]
```

Reply:
Mentions are added.